site/app/libraries/database/DatabaseUtils.php

```diff
@@ -4,9 +4,18 @@
 
 namespace app\libraries\database;
 
+use app\libraries\DateUtils;
+
 class DatabaseUtils {
     public static function formatQuery($sql, $params): string {
+        if (is_null($params)) {
```

Comment:
How do we hit this case? When/how is `$params` null?

Reply:
I am unsure.  I fixed the issue with DateTime objects and another error about null `$params` popped up and I couldn't find a source of the issue so I just wrapped it in the `is_null` and moved on.

Reply:
This was a thrown exception?

Reply:
This was the error: <img width="661" alt="Untitled" src="https://user-images.githubusercontent.com/16820599/141522233-f678e469-dccf-4044-94ff-78cb3c79d4f4.png">
(line 15 is the start of the for-loop in this example)